test/0.4.24/lido-deposit-scenarios.test.js

```diff
@@ -4,22 +4,44 @@ const { EvmSnapshot, setBalance, getBalance } = require('../helpers/blockchain')
 const { ZERO_ADDRESS } = require('../helpers/constants')
 const { assert } = require('../helpers/assert')
 const { wei } = require('../helpers/wei')
-const { StakingModuleStub } = require('../helpers/stubs/staking-module.stub')
 const { PUBKEY_LENGTH, FakeValidatorKeys, SIGNATURE_LENGTH } = require('../helpers/signing-keys')
-const { GenericStub } = require('../helpers/stubs/generic.stub')
+const { ContractStub } = require('../helpers/contract-stub')
```

Comment:
The file is missing in this PR :eyes:

Reply:
Oops, forgot to include it in the initial commit, sorry. Added it